frontend/src/components/LandingPage/FeaturedElection.tsx

```diff
@@ -0,0 +1,17 @@
+import { Paper, Typography } from "@mui/material";
+import React from 'react';
+
+export default ({electionId}) => {
+    return <Paper className='featuredElection' elevation={8} sx={{
```

Comment:
Try out the Material UI card component for these. It uses the paper component but with extra features.
https://mui.com/material-ui/react-card/